### ISSUE TITLE
make SSL default for website / redirect http->https

### ISSUE THREAD

**Issue by jeensg** [abrensch/brouter#311]
I noticed that brouter.de doesn't redirect to https automaticalley, although it's working. Is it possible to implement redirection?

**Comment by poutnikl** [comment]
I guess it could interfere with applications that rely on plain http.....

**Comment by jeensg** [comment]
Mh, I get that. But I don't think that it should be supported anymore. Are there any types of applications that really are dependent on that. Or is it just the legacy-support, beause they are not able / don't want to update!?
Anyway, it's not soooo important, I'm always only wondering how we cannot implement this by now.

Thanks for the advices, already using these ways :-)

**Comment by poutnikl** [comment]
wget on termux on Android does support Https, so I have no objection against dropping plain http support. :-) Let wait for the webmaster.

**Comment by abrensch** [comment]
>> Let wait for the webmaster.

ja, I can do that. It's an nginx-config. After installing the (LetsEncrypt) SSL-Certificate that the renewal-process is such stable that I really can rely on it. But it seems to work very stable.

**Comment by utagawal** [comment]
Hi any plans to deply this feature in the next release ?
